### ISSUE TITLE
AStar32U4Motors Reverse not working on A*32U4 Prime LV Booard

### ISSUE THREAD

**Issue by verrol** [pololu/a-star-32u4-arduino-library#1]
I can't get my motors to reverse using this library on my Prime LV board. Negative values don't reverse, neither does the flipMx() method. Additionally, the example sketch has this in the setup() method:

  // Uncomment to flip a motor's direction:
  //motors.flipLeftMotor(true);
  //motors.flipRightMotor(true);

But those methods don't exist in the library.

**Comment by kevin-pololu** [comment]
Hello, verrol.

Those lines should say:

```
// Uncomment to flip a motor's direction:
//motors.flipM1(true);
//motors.flipM2(true);
```

Thanks for pointing out the mistake; I'll fix the example soon.

At the moment, the AStar32U4Motors class is primarily intended for use with an [A-Star 32U4 Robot Controller](https://www.pololu.com/product/3116), although it should work with an A-Star Prime if you connect appropriate motor drivers in the same way. Specifically, they should use the pins listed in the "Motor drivers" section of the [A-Star Robot Controller user's guide](https://www.pololu.com/docs/0J66). What motor drivers are you using and how do you have them connected?

**Comment by verrol** [comment]
Hi Kevin,

Thanks for your prompt response.

I both the assembled kids and board. It whatever motor driver that comes form Pololu.

Sorry, that wasn't much help. Is there somewhere I can look to give you specifics?

Thanks,

.v

**Comment by kevin-pololu** [comment]
It doesn't sound like a problem with the library is causing you to still have trouble, so please continue troubleshooting with us through our regular support channels: either by posting on [our forum](http://forum.pololu.com/) or by [emailing us directly](https://www.pololu.com/contact). Please show us some pictures of your setup so we can see what parts you are using and how you have everything connected.

The motor flipping code is fixed in the Motors example now (in release 1.0.1), so I will close this issue.
